src/script/all.js

```diff
@@ -0,0 +1,203 @@
+import { renderTemplate, request } from './helpers';
+
+const numPerPage = 10;
+let currentStartProduct = 0;
+let activePagination = 1;
```

Comment:
```suggestion
let currentPage = 1;
```
Còn cái trên currentStartProduct là gì em nhỉ?